base_comment_template/readme/CONTRIBUTORS.rst

```diff
@@ -0,0 +1,14 @@
+* Xavier Jimenez <xavier.jimenez@qubiq.es>
+* Nicolas Bessi <nicolas.bessi@camptocamp.com>
+* Yannick Vaucher <yannick.vaucher@camptocamp.com>
+* Guewen Baconnier <guewen.baconnier@camptocamp.com>
+* Simone Rubino <simone.rubino@agilebg.com>
+* Simone Rubino <simone.rubino@agilebg.com>
```

Comment:
This line is duplicated

Reply:
I've remove the duplicated line and squash last commits